src/main/webapp/app/exam/manage/exams/exam-checklist-component/exam-checklist.component.ts

```diff
@@ -63,6 +89,14 @@ export class ExamChecklistComponent implements OnChanges, OnInit, OnDestroy {
                 !!this.exam.numberOfExamUsers && this.exam.numberOfExamUsers > 0 && this.examChecklistService.checkAllExamsGenerated(this.exam, this.examChecklist);
             this.numberOfStarted = this.examChecklist.numberOfExamsStarted;
             this.numberOfSubmitted = this.examChecklist.numberOfExamsSubmitted;
+            if (this.isExamOver) {
+                if (this.examChecklist.numberOfTotalExamAssessmentsFinishedByCorrectionRound) {
```

Comment:
Could this make problems with completely automatically assessed exams? Maybe check explicitly for undefined

Reply:
Makes sense, I am adding this